charts/ingress-nginx/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: ingress-nginx
-version: 2.4.0
+version: 2.5.0
```

```suggestion
version: 2.6.0
```
Another change actually got in before yours. Bump the minor version and squash.

---

Ok, bumped and squashed one more time.